packages/InlineEditing/src/index.js

```diff
@@ -0,0 +1,2 @@
+export { default } from "./Editor";
+export { default as Table } from "./Table";
```

Comment:
You're not exposing the `<Input>`, `<ListBox>` or `<Textarea>` for consumers? 

Will there eventually be a need to have inline editable `<Select>` or `<DataPicker>` types?

Reply:
I don't like importing from `lib`.  It seems like a back door – I'd rather see a clear offering of the API in the `index` entry point.

Reply:
you don't want to load all the widget if you want to use just one, this is a component that has many "plugable" components, there is not a reason to load all of them from the index. Think about this like the icons, you can't load all the icons just to use one.